.github/workflows/release-installer.yml

```diff
@@ -0,0 +1,14 @@
+name: Trigger installer release
+
+on:
```

Comment:
I think this triggers too early. It needs to trigger after all binaries have been uploaded.

Reply:
I've modified this to run after the goreleaser workflow.